packages/components/src/Button/PaginationLink/PaginationLink.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import { GenericProps, GenericButton } from "../GenericButton"
+
+export interface PaginationLinkProps extends GenericProps {
+  pageNumber: number
+  isActive?: boolean
+  /* A description for screen readers to understand because label used here is a number */
+  "aria-label": string
+}
```

Comment:
```suggestion
export type PaginationLinkProps = GenericButtonProps & {
  pageNumber: number
  isActive?: boolean
  /* A description for screen readers to understand because label used here is a number */
  "aria-label": string
}
```